src/app/static/src/app/components/load/UploadNewProject.vue

```diff
@@ -1,8 +1,8 @@
 <template>
     <div id="load-project-name">
         <modal id="load" :open="openModal">
-            <label class="h5" for="project-name-input" v-translate="'enterProjectName'"></label>
-            <input id="project-name-input"
+            <label class="h5" :for="inputId" v-translate="'enterProjectName'"></label>
+            <input :id="inputId"
```

Comment:
Had to set this ID here to avoid chrome dev console showing an error for duplicate input field ID (for upload zip and upload json forms)

Reply:
Perhaps there is a better way to achieve this though?

Reply:
i think passing it as a prop is completely fine!